text_generation/causal_lm/cpp/continuous_batching/library/src/continuous_batching_pipeline.cpp

```diff
@@ -74,6 +83,10 @@ class ContinuousBatchingPipeline::Impl {
         m_requests.erase(new_end, m_requests.end());
     }
 
+    void _free_all_requests() {
+        m_requests.erase(m_requests.begin(), m_requests.end());
```

Comment:
maybe we extend `_free_finished_requests` to free all non_running_requests ? and rename to `_free_non_running_requests` 

In this case we will have unified method

Reply:
Yes, moved this cycle to `schedule()` method.

Reply:
Done.